Makefile

```diff
@@ -36,3 +36,6 @@ test:
 .PHONY: testacc
 testacc:
 	TF_ACC=1 go test ./... -json -v $(TESTARGS) -count 1 -timeout 120m | gotestfmt
+
+lint:
+	docker run --rm -v "$(shell pwd):/app" -w /app golangci/golangci-lint:v1.64.8 golangci-lint run --fix
```

Comment:
IMO, forcing the use of docker here doesn't make sense (it assumes too much). Let's just run `golangci-lint` directly (it should be a safe assumption that the user has it, as it's a very common tool if you are doing go development. Locally if you personally don't want to install it, you could alias the `golangci-lint` tool to the docker command above.

Reply:
I do not have it, I tried installing it and ran into v2 issues regarding my OS, hence this workaround.